src/CMakeLists.txt

```diff
@@ -199,6 +199,12 @@ add_library(parthenon
   parthenon_mpi.hpp
 )
 
+add_library(parthenon "${parthenon_sources}")
```

Comment:
Now that you've reverted the `GLOB`s here, put the sources back directly into the `add_library` call.